exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/columnreaders/NullableFixedByteAlignedReaders.java

```diff
@@ -132,6 +137,9 @@ protected void readField(long recordsToReadInThisPass) {
           valueVec.getMutator().setSafe(valuesReadInCurrentPass + i, getDateTimeValueFromBinary(binaryTimeStampValue));
         }
       }
+      // The nanos precision is cut to millis. Therefore the length of single timestamp value is 8 bytes(s)
+      // instead of 12 byte(s).
+      dataTypeLengthInBits = timestampLengthInBits;
```

Comment:
Just trying to understand this better: Is it OK to set the length to 64 bits even if PARQUET_READER_INT96_AS_TIMESTAMP is set to false?